website/guides/sgb_border.md

```diff
@@ -0,0 +1,177 @@
+
+```
+superfamiconv -v -i input.png -p output.pal -t output.4bpp -P 4 -m output.pct -M snes --color-zero 0000ff -B 4
+```
+
+- `--color-zero` should be the color _You_ used for transparency, in my case it was blue.
+  - it can also be set to `00000000` to use the actual transparent color, however, this may cause some issues..
+- `-v` is optional
+- You can add a row of the transparent color at the top of the image to force superfamiconv to make it tile #0, then `incbin "output.pct", 64` to leave out that row.
+- `-P 4` sets the base palette to the 4th one, and **SGB borders use SNES palettes 4, 5, and 6.** as of writing this, this option only works if You built superfamiconv from source.
+
+## Uploading
+
+As stated before, the SGB border consists of tile data, picture data, and palette data. these are split across 2-3 packets:
+
+- `CHR_TRN` (`$99`) is used to send 4KiB of tile data.
+
+  - since the border can use up to 8KiB of tiles, bit 0 of the second byte specifies which "half" Youre sending
+    - `$99, $00` if the screen is loaded with the first 4KiB of tile data
+    - `$99, $01` if the screen is loaded with the second 4KiB of tile data
+
+- `PCT_TRN` (`$a1`) is used to send the picture and palette data. it also swaps the border, generally a good idea to send it after the tile data[^1]
+  - assuming tiles `0`-`255` use VRAM from `$8000` to `$8fff`:
+    - the picture data must be at `$8000`-`$873f` (last 64 bytes are _usually_ offscreen, see [notes](#notes))
+    - palette data must be at `$8800`-`$885f`
+    - everything else is ignored
+      - how You skip putting data at `$8740`-`$87ff` is up to You, I prefer doing separate copies, valen prefers copying tilemap and palette data in one go, with the area between them padded with `$ff`.
+
+If this makes no sense You could also read [pandocs](https://gbdev.io/pandocs/SGB_Command_Border.html)
+
+[^1]: You can send a `CHR_TRN` up to \~60 frames after the `PCT_TRN` for it to apply to the current border, but not all emulators will emulate this. its fine to just pretend `CHR_TRN`s must go before `PCT_TRN`
+
+## Notes
+
+1. You can set the first row of tiles to Your transparent color to force superfamiconv to put the transparent tile as the 1st tile, however You must then exclude 64 bytes of the tilemap (`incbin "border.pct"` -> `incbin "border.pct", 64`)
+
+2. You can use palettes 0-3 and 7, if You really know what Youre doing (animated borders yay), but You will probably have to edit the border in YY-CHR, as there arent really any other tools for that
+
+3. when the SNES lags, scanline 225 of the SGB border will be visible! You can set the topmost row of the 29th row of tiles to black to hide this
+
+4. if this doesnt work for You, or something here is unclear, then please contact me, preferably via discord (`@zlago`) or mastodon (`@zlago@mastodon.gamedev.place`) as I tend to check those reasonably often
```

Comment:
we should just redirect to discord here (link to the chat channels on the gbdev.io website)